Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c

```diff
@@ -676,6 +678,14 @@ PlatformUpdateAcpiGnvs (
   if (PseCanPciMmBase != 0xFFFF) {
     PchNvs->PseCan1Enabled                        = 1;
   }
+  if (SiCfgData != NULL) {
+    if ((UINT8)SiCfgData->EnableTimedGpio0 != 0){
+      PchNvs->EnableTimedGpio0 = 1;
+    }
+    if ((UINT8)SiCfgData->EnableTimedGpio1 != 0){
+      PchNvs->EnableTimedGpio1 = 1;

```

Comment:
should you fully control the time gpios by the config data as below?
if (SiCfgData != NULL) {
PchNvs->EnableTimedGpio0 = SiCfgData->EnableTimedGpio0;
PchNvs->EnableTimedGpio1 = SiCfgData->EnableTimedGpio1;
}

Reply:
Noted and submitted patch